src/components/Sing/ScoreSequencer.vue

```diff
@@ -308,256 +300,329 @@ export default defineComponent({
       });
     });
     const scrollBarWidth = ref(12);
-
     const sequencerBody = ref<HTMLElement | null>(null);
-
-    // ノートの追加
-    const addNote = (event: MouseEvent) => {
-      const eventOffsetBaseX = event.offsetX / zoomX.value;
-      const eventOffsetBaseY = event.offsetY / zoomY.value;
-      const positionBaseX =
-        gridCellBaseWidth.value *
-        Math.floor(eventOffsetBaseX / gridCellBaseWidth.value);
-      const position = baseXToTick(positionBaseX, tpqn.value);
-      const noteNumber = baseYToNoteNumber(eventOffsetBaseY);
-      if (noteNumber < 0) {
-        return;
+    // プレビュー
```

Comment:
ちょっとここにFIXMEコメントだけ書かせていただけると･･･！
```suggestion

    // プレビュー
    // FIXME: 関連する値を１つのobjectにまとめる
```